### PR TITLE
update category-payment-ir

### PR DESCRIPTION
**Pull request by alirezax5**

### CODE REVIEW

**Review by rootmelo92118 — APPROVED** [review]

File: data/category-payment-ir [hunk 1]
```diff
@@ -5,10 +5,12 @@ asanpardakht.net
...
+pay.ir
 payping.ir
 pec.ir
 pep.co.ir
 sadadpsp.ir
 sep.ir
 shaparak.ir
-zarinpal.ir
+weswap.digital
+zarinpal.com
```
Comment (IceCodeNew) [inline data/category-payment-ir:16]: please note there should be an empty line at the end of the file.
@alirezax5 @rootmelo92118

File: data/category-payment-ir [hunk 2]
```diff
@@ -5,10 +5,12 @@ asanpardakht.net
...
 iranicard.ir
 mydigipay.com
+pay.ir
 payping.ir
 pec.ir
 pep.co.ir
 sadadpsp.ir
 sep.ir
 shaparak.ir
-zarinpal.ir
```
Comment (IceCodeNew) [inline data/category-payment-ir:14]: BTW, why is this domain been deleted?

Comment (alirezax5) [reply data/category-payment-ir:14]: the main domain is zarinpal.com

Comment (alirezax5) [reply data/category-payment-ir:14]: zarinpal does not use this domain (zarinpal.ir) anymore and the host is not set on it

https://who.is/dns/zarinpal.ir

photo_2023-06-07_16-13-18

Comment (IceCodeNew) [reply data/category-payment-ir:14]: The DNS record does not matter here, this domain is still active.
You said zarinpal does not use this domain (zarinpal.ir) anymore, is there any source mentioned about that?

Comment (alirezax5) [reply data/category-payment-ir:14]: > The DNS record does not matter here, this domain is still active. You said zarinpal does not use this domain (zarinpal.ir) anymore, is there any source mentioned about that?

I didn't see Zarinpal using the .ir domain anymore, but now if you think it should be in the list, I will add it

**Pull request merged**